internal/fields/validate.go

```diff
@@ -212,7 +211,17 @@ func parseElementValue(key string, definition FieldDefinition, val interface{})
 
 	var valid bool
 	switch definition.Type {
-	case "date", "ip", "constant_keyword", "keyword", "text":
+        case "constant_keyword", "keyword":
```

Comment:
I suspect we may rerun `make format` for this file.

Reply:
Done